### PR TITLE
Script improvements resiliency

### PR DESCRIPTION
**Pull request by matskiv**
# Description
Adding some improvements and a new script for the perf testing work.
cc @trepel @austincunningham 

Example output of the new script:
https://gist.githubusercontent.com/matskiv/c294b37b41102bf16329c914bac1605e/raw/7127a4df520a4845c53bc1fa70cc2cb987c069b0/output.txt

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: scripts/capture_resource_metrics.sh [hunk 1]
```diff
@@ -79,20 +80,32 @@ LOAD_QUERIES=(\
...
 for query in "${IDLE_QUERIES[@]}";
 do
-  curl -s -G -H "Authorization: Bearer $TOKEN" --data-urlencode "query=$query" --data-urlencode "time=$startTime" -H 'Accept: application/json' $PROM_QUERY_ROUTE | jq -r ".data.result[0].value[1]"
+  runQuery "$query" "$startTime"
 done
 
 for query in "${LOAD_QUERIES[@]}";
 do
-  curl -s -G -H "Authorization: Bearer $TOKEN" --data-urlencode "query=$query" --data-urlencode "time=$endTime" -H 'Accept: application/json' $PROM_QUERY_ROUTE | jq -r ".data.result[0].value[1]"
+  runQuery "$query" "$endTime"
```
Comment (austincunningham) [inline scripts/capture_resource_metrics.sh:110]: verified  output ```/capture_resource_metrics.sh
60
36
27
229.88712310791016
137.8338394165039
127.9549331665039
11.353000000000005
23670.673217773438
9906.188281249993
70.0390625
1805.0022569444443
1167.079644097222
1512.7473958333335
15146.548437500003
0.33211507670322427
0.0025006061221629033
0.04032196133929438
0.012566208266896221
0.17896114323686715
0.708667354581686
9806.83984375
70.37890625
1801.20703125
1178.9140625
1633.82421875
15104.109375
1.5375191645367758
0.10416336774832513
1.2174912670907152
0.028607494722003628
0.39042682864224737
3.42967602103125
```

File: scripts/hyperfoil-all-means.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    for percentile in "${means[@]}"
+    do
+      mean=$(cat /tmp/hf-tmp-stats-data.json | jq -r "select(.phase | test(\"steady\")) | select(.metric| test(\""${type}"\")) | .total.summary | \"\(.percentileResponseTime.\"${percentile}\" / 1000000) \(.responseCount)\"" | awk '{w = w + $2; e = e + $1 * $2;} END {printf "%.3f", e/w}')
+      echo -ne "${mean}\t"
+    done
+  done
+
+  curl -s -H "$HEADER" ${HYPERFOIL_URL}/run/${RUN} | jq -r '"\(.benchmark)\t\(.description)"'
+done
+echo ""
```
Comment (austincunningham) [inline scripts/hyperfoil-all-means.sh:49]: verified 
```bash
./hyperfoil-stat-multiple.sh 0x0001 0x0002
RUN #   get     50.0    90.0    99.0    99.9    99.99   post    50.0    90.0    99.0    99.9    99.99   Benchmark       Description
0001            13.650  22.207  66.585  115.194 147.550         24.977  35.946  78.086  123.273 150.471 rhsso_tokens_py_payloads        null
0002            13.539  22.828  65.313  113.325 180.517         23.993  34.421  77.462  128.711 222.953 rhsso_tokens_py_payloads        null
```

**Review by austincunningham — APPROVED** [review]
/lgtm
/approve

**Comment by matskiv** [comment]
Thanks Austin!
/approve

**Pull request merged**